backend/src/election/mod.rs

```diff
@@ -90,3 +91,43 @@ pub async fn election_status(
     let statuses = polling_station_repo.status(id).await?;
     Ok(Json(ElectionStatusResponse { statuses }))
 }
+
+/// Download a generated PDF
+#[utoipa::path(
+    get,
+    path = "/api/elections/{election_id}/download_pdf",
+    responses(
+        (
+            status = 200,
+            description = "PDF",
+            content_type="application/pdf",
+            headers(
+                ("Content-Type", description = "text/pdf; charset=utf-8"),
```

Comment:
You forgot to change the content type here.

Reply:
d'oh! Fix: [38f4c30](https://github.com/kiesraad/abacus/pull/262/commits/38f4c30153929caafd44a9022ee6ae5d2aec984c)